package.json

```diff
@@ -87,5 +85,9 @@
     "re:build": "rescript",
     "re:watch": "rescript clean && rescript build -w"
   },
-  "packageManager": "yarn@4.1.1"
+  "packageManager": "yarn@4.1.1",
+  "devDependencies": {
+    "vite": "^5.0.0",
+    "vite-plugin-ruby": "^5.0.0"
+  }
```

Comment:
We had dropped the idea of maintaining `devDependencies` & `dependencies` on our packages long back; Lets keep it as a flat list.

Reply:
The reasoning was that these packages are installed only for development / building, and aren't kept around in the production image.